packages/selectInput/components/SelectInput.tsx

```diff
@@ -0,0 +1,181 @@
+import * as React from "react";
+import { cx } from "emotion";
+import { select, selectIcon, selectContainer } from "../style";
+import {
+  inputContainer,
+  inputAppearances,
+  getLabelStyle
+} from "../../shared/styles/formStyles";
+import {
+  inputReset,
+  padding,
+  visuallyHidden,
+  display
+} from "../../shared/styles/styleUtils";
+import Icon from "../../icon/components/Icon";
+import { SystemIcons } from "../../icons/dist/system-icons-enum";
+import { iconSizeXs } from "../../design-tokens/build/js/designTokens";
+import FormFieldWrapper from "../../shared/components/FormFieldWrapper";
+
+export enum SelectInputAppearance {
```

Comment:
Can we also share this with `TextInput`?

Reply:
Good idea. I just have to figure out the right place to put it 🤔